src/main/java/com/cen4010/bookstore/book/Book.java

```diff
@@ -3,20 +3,20 @@
 import com.cen4010.bookstore.profileManagement.entity.UserEntity;
 import com.cen4010.bookstore.wishlist.WishList;
 import com.fasterxml.jackson.annotation.JsonIgnore;
+import org.hibernate.annotations.Type;
+
 import java.util.HashSet;
 import java.util.Set;
 import java.util.UUID;
-import javax.persistence.Column;
-import javax.persistence.Entity;
-import javax.persistence.Id;
-import javax.persistence.ManyToMany;
-import org.hibernate.annotations.Type;
+import javax.persistence.*;
 
 
 @Entity
+@Table
```

Comment:
What's the purpose of this annotation? For that matter, what's the purpose of making any modifications to the book entity? It's been working fine.